src/devices/BrickPi3/Movement/Vehicle.cs

```diff
@@ -171,7 +171,7 @@ private void RunMotorSyncTime(BrickPortMotor[] ports, int[] speeds, int timeout)
             }
 
             // create a timer for the needed time to run
-            if (_timer == null)
+            if (_timer is null)
             {
                 _timer = new Timer(RunUntil, ports, TimeSpan.FromMilliseconds(timeout), Timeout.InfiniteTimeSpan);
```

Comment:
Is _timer ever disposed/nulled? It is an `IDisposable` object and should be handled (which means this class should also be `IDisposable`